src/utils/saga/toasts.js

```diff
@@ -0,0 +1,55 @@
+import { put } from 'redux-saga/effects';
+
+import { actions as toastActions } from 'reducers/toast';
+/**
+ * Custom Toast Helper
+ * @param object msg
+ * @example
+ * { title: 'Toast Title' (optional),
+ *   icon: 'times-square' (optional),
+ *   message: 'My response message' (required) }
+ */
+export function* toast(msg) {
+  const { icon, title, message } = msg;
+  const response = {};
+
+  response.title = title ? title : 'Server Message';
```

Comment:
Why not initialize these inline with the object above?

Reply:
fixed